marlowe-playground-client/src/Language/Javascript/MarloweJS.ts

```diff
@@ -1,5 +1,7 @@
 import bignumber = require("bignumber.js")
 
+const MAX_SAFE_INTEGER = 9007199254740991;
+
```

Comment:
You can use `Number.MAX_SAFE_INTEGER` directly, I think

Reply:
Oh, you are right, I thought I would need to change the imports, but it just works